### PR TITLE
10.18.0.59

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-18-0/devices-10-18-0.md [hunk 1]
```diff
@@ -198,6 +216,55 @@ Build version</th>
...
+<td>LWM2M</td>
+<td>In certain cases, the registration couldn't be associated to a LWM2M 1.1 SEND request, leading to a 4.04 CoAP response. This is now fixed.</td>
+<td>DM-2270</td>
+<td>10.18.0.47</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>The LWM2M Agent now ignores trailing commas at the end of object links in the registration request of a LWM2M client.</td>
```
Comment (BeateRixen) [inline content/release-10-18-0/devices-10-18-0.md:245]:
```suggestion
<td>The LWM2M agent now ignores trailing commas at the end of object links in the registration request of a LWM2M client.</td>
```

File: content/release-10-18-0/applications-10-18-0.md [hunk 2]
```diff
@@ -306,6 +316,38 @@ Build version</th>
...
+<td>Java SDK</td>
+<td>The dependency to "com.jayway.jsonpath:json-path" has been removed from the Java SDK.</td>
+<td>MTM-53341</td>
+<td>10.18.0.59</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>Java SDK</td>
+<td>The dependency to "org.json:json" has been removed from the Java SDK.</td>
```
Comment (rssag) [inline content/release-10-18-0/applications-10-18-0.md:329]:
```suggestion
<td>The dependency on "org.json:json" has been removed from the Java SDK.</td>

```

File: content/release-10-18-0/devices-10-18-0.md [hunk 3]
```diff
@@ -198,6 +216,55 @@ Build version</th>
...
+<td>LWM2M</td>
+<td>Bulk device registrations and other operations being executed on the LWM2M connector device now show the status FAILED if a problem occurs. Prior to this change, partial failures were reported as SUCCESSFUL.</td>
+<td>DM-1951</td>
+<td>10.18.0.54</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>In certain cases, the registration couldn't be associated to a LWM2M 1.1 SEND request, leading to a 4.04 CoAP response. This is now fixed.</td>
```
Comment (rssag) [inline content/release-10-18-0/devices-10-18-0.md:237]:
```suggestion
<td>In certain cases, the registration couldn't be associated with a LWM2M 1.1 SEND request, leading to a 4.04 CoAP response. This is now fixed.</td>
```

File: content/release-10-18-0/devices-10-18-0.md [hunk 4]
```diff
@@ -198,6 +216,55 @@ Build version</th>
...
+<td>LWM2M</td>
+<td>The LWM2M agent is now able to properly convert the timestamps from the SenML data reported by the LWM2M client to a platform compatible date-time format for performing respective resource actions.</td>
+<td>DM-2150</td>
+<td>10.18.0.43</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>Multi-line LWM2M post-operations were not executed right after the LWM2M device's new registration when realtime was disabled for the tenant who owns the device. As a result the device might not receive these operations until the next LWM2M device's registration update. This issue is now resolved and LWM2M post-operations are executed right after the LWM2M device's new registration, no matter whether realtime is enabled or not for this kind of devices.</td>
```
Comment (rssag) [inline content/release-10-18-0/devices-10-18-0.md:261]: not sure about the post-operation (i recall that i commented on this in another PR yesterday)

File: content/release-10-18-0/applications-10-18-0.md [hunk 5]
```diff
@@ -306,6 +316,38 @@ Build version</th>
 <td>UI</td>
 </tr>
 
+<tr>
+<td>Java SDK</td>
+<td>The dependency to "com.jayway.jsonpath:json-path" has been removed from the Java SDK.</td>
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-18-0/applications-10-18-0.md:321]: I was also wondering about the formatting here. Is this a file or a path? Then italics would probably be correct.

[the remark this answers is not in the document]
Comment (rssag) [reply content/release-10-18-0/applications-10-18-0.md:321]: looks like a dot-notation reference to a specific class in a library. i'm not sure about the colon though

**Pull request merged**